server/integration/testsuite/pom.xml

```diff
@@ -269,6 +307,9 @@
         <node1.ip>127.0.0.1</node1.ip>
         <node2.ip>127.0.0.1</node2.ip>
         <node3.ip>127.0.0.1</node3.ip>
+        <node0.mgmt.port>9990</node0.mgmt.port>
+        <node1.mgmt.port>10090</node1.mgmt.port>
+        <node2.mgmt.port>10190</node2.mgmt.port>
```

Comment:
node3 is only used in tests for rolling upgrades. These are not run as part of domain mode tests. We can add it later when we modify rolling ugprade tests for domain mode.